src/neo/SmartContract/ContractParameterType.cs

```diff
@@ -10,6 +10,7 @@ public enum ContractParameterType : byte
         ByteArray = 0x05,
         PublicKey = 0x06,
         String = 0x07,
+        Null = 0x08,
```

Comment:
We don't need `Null` in `ContractParameterType`.

Reply:
Now we get JSON `{ "type": "Any"}` when the `StackItem` is Null in `ApplicationLog`. 
We won't support `ContractParameter` to `StackItem` for `ContractParameterType.Any` and `ContractParameterType.InteropInterface`. I will just use `ContractParameter` type in `RpcApplicationLog`.